dashboard/app/views/levels/_form.html.haml

```diff
@@ -42,7 +42,12 @@
   .field
     = f.hidden_field :published
   .actions
-    = f.submit 'Save and publish your completed work', {:id => 'publishLevel', :style => "background-color: #f0c14b"}
+    = f.submit 'Save and publish your completed work', {:class => 'publishLevel', :style => "background-color: #f0c14b; margin-bottom: 10px"}
+    %p Or, save, publish, and return to:
+    %ul
+      - @level.script_levels.each do |script_level|
```

Comment:
Note that this hits the DB. Though totally worth it, I do want to call that tout.